opencog/atoms/pattern/PatternLink.cc

```diff
@@ -589,7 +589,7 @@ void PatternLink::unbundle_virtual(const std::set<Handle>& vars,
 		bool is_virtual = false;
 		bool is_black = false;
 
-#ifdef BORKEN_DOESNT_WORK
+#ifdef BROKEN_DOESNT_WORK
```

Comment:
well, if it merely broken, then its not borken.  (c.f. Robert Bork, who was borked.)